src/Component/src/Metadata/Resource/ResourceClassList.php

```diff
@@ -14,11 +14,11 @@
 namespace Sylius\Resource\Metadata\Resource;
 
 /**
- * A collection of resource names.
+ * A list of resource class names.
  *
  * @experimental
  */
-final class ResourceNameCollection implements \IteratorAggregate, \Countable
+final class ResourceClassList implements \IteratorAggregate, \Countable
 {
     /**
      * @param string[] $names
```

Comment:
```patch
---     public function __construct(private readonly array $names = [])
+++     public function __construct(private readonly array $classNames = [])
+++     public function __construct(private readonly array $classes = [])
```

Reply:
`src/Component/src/Metadata/Resource/Factory/AttributesResourceClassListFactory.php` uses `$classes` internally, so I guess it should be preferred.